dev-util/wsta/wsta-0.4.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="A CLI development tool for WebSocket APIs"
+HOMEPAGE="https://github.com/esphen/wsta/"
+SRC_URI="https://github.com/esphen/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-3"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="dev-libs/openssl:*"
```

Comment:
Just had a look at this. It will not build as-is with libressl because [cargo depends on openssl](https://github.com/gentoo/gentoo/blob/master/dev-util/cargo/cargo-0.13.0-r1.ebuild#L103)